cmd/rqlite/main.go

```diff
@@ -18,20 +18,22 @@ import (
 	"github.com/Bowery/prompt"
 	"github.com/mkideal/cli"
 	"github.com/rqlite/rqlite/cmd"
+	httpcl "github.com/rqlite/rqlite/cmd/rqlite/http"
 )
 
 const maxRedirect = 21
 
 type argT struct {
 	cli.Helper
-	Protocol    string `cli:"s,scheme" usage:"protocol scheme (http or https)" dft:"http"`
-	Host        string `cli:"H,host" usage:"rqlited host address" dft:"127.0.0.1"`
-	Port        uint16 `cli:"p,port" usage:"rqlited host port" dft:"4001"`
-	Prefix      string `cli:"P,prefix" usage:"rqlited HTTP URL prefix" dft:"/"`
-	Insecure    bool   `cli:"i,insecure" usage:"do not verify rqlited HTTPS certificate" dft:"false"`
-	CACert      string `cli:"c,ca-cert" usage:"path to trusted X.509 root CA certificate"`
-	Credentials string `cli:"u,user" usage:"set basic auth credentials in form username:password"`
-	Version     bool   `cli:"v,version" usage:"display CLI version"`
+	Alternatives string `cli:"a,alternatives" usage:"comma separated list of 'host:port' pairs to use as fallback"`
```

Comment:
So I've never really liked the `Host` and `Port` parameters the CLI currently takes, I much prefer the version you choose. But I wasn't going to change it until the next major version. As it happens, I'm going to release v7 soon (the PR is in progress at https://github.com/rqlite/rqlite/pull/957), so this gives us a chance to fix up the CLI.

What do you think about just reworking the CLI options so that there is a single `host` parameter which takes a comma-delimited list of scheme:host:port values? E.g. 

`-host=http://localhost:4001,http://localhost:4003` (or maybe `-hosts`?)

If the scheme wasn't supplied, HTTP would be assumed. We would remove the existing `Protocol` parameter too. All other parameters would remain. It might make your code clearer too.

WDYT? I would make your updates part of the upcoming 7.0 release.

Reply:
I didn't want to remove the Host: Port initially for backward compatibility reasons, I would be down for changing that to use a list of hosts to connect, but I prefer to do that as part of a separate PR.